target/linux/ramips/image/rt305x.mk

```diff
@@ -1086,6 +1086,20 @@ define Device/unbranded_a5-v11
 endef
 TARGET_DEVICES += unbranded_a5-v11
 
+define Device/tenda_a6
```

Comment:
Please relocate due to alphabetic sorting.

Reply:
Roger